ts/Core/Chart/Chart.ts

```diff
@@ -3383,9 +3382,7 @@ class Chart {
         if (updateAllSeries) {
             chart.getSeriesOrderByLinks().forEach(function (series): void {
                 // Avoid removed navigator series
-                if (series.chart) {
-                    series.update({}, false);
-                }
+                series.update({}, false);
```

Comment:
Why was the check for `series.chart` removed? What consequences may it have? The comment (which is now dangling) describes a removed navigator series.

Reply:
Sorry, I rushed with it. I saw an unusual `if` check, I thought it was unnecessary and removed it. Don't know the consequences :(
Already restored.